features/margin.yml

```
@@ -0,0 +1,15 @@
+name: margin
+description: Margins define space surrounding a box. The `margin` CSS property provides a shorthand for `margin-top`, `margin-right`, `margin-bottom`, and `margin-left`.
```

The styleguide says-
"Start descriptions with words that are distinct to the feature. For example, prefer "The some-prop CSS property…" and avoid "The CSS property some-prop…."

See also [defines](https://github.com/web-platform-dx/web-features/blob/main/docs/guidelines.md#defines).

```suggestion
description: The `margin` CSS property sets space surrounding a box, and is a shorthand for `margin-top`, `margin-right`, `margin-bottom`, and `margin-left`.
```
I'm wondering if "a box" is descriptive enough. Maybe `an element's box`, or maybe `surrounding a box, outside its border`?